src/everest/export.py

```diff
@@ -162,7 +162,9 @@ def get_internalized_keys(config: EverestConfig, batch_ids: Optional[Set[int]] =
     with open_storage(config.storage_dir, "r") as storage:
         for batch_id in batch_ids:
             case_name = f"batch_{batch_id}"
-            experiment = storage.get_experiment_by_name(f"experiment_{case_name}")
+            experiments = [*storage.experiments]
+            assert len(experiments)
+            [experiment] = experiments
```

Comment:
I do not find this very clear code, why not just `experiment = next(storage.experiments)`? An empty `storage.experiments` would raise an exception, which should indicate a bug.

Also the assert should probably be `== 1`?